readthedocs/templates/builds/versionautomationrule_list.html

```diff
@@ -0,0 +1,48 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Automation Rules" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block project-automation-rules-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Automation Rule" %}{% endblock %}
+
+{% block project_edit_content %}
+  <p>Run specific actions when a new version is added.</p>
```

Comment:
This paragraph should be expanded a little more and link to the the docs for more information probably.

Reply:
I don't think adding more words here is the best, a link to the docs should be better. #6059.